packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -172,11 +172,19 @@ export class H5GroveApi extends ProviderApi {
     }
 
     if (isDatasetResponse(response)) {
-      const { attributes: attrsMetadata, dtype, shape } = response;
+      const {
+        attributes: attrsMetadata,
+        dtype,
+        shape,
+        chunks,
+        filters,
+      } = response;
       const attributes = await this.processAttrsMetadata(path, attrsMetadata);
 
       return {
         ...baseEntity,
+        chunks: chunks ? chunks : undefined,
+        filters: filters ? filters : undefined,
```

Comment:
Ideally, I would prefer the fields to be absent if `null`.

I don't know if there is a better way to do this ?